lib/IDE/CodeCompletion.cpp

```diff
@@ -4137,9 +4137,13 @@ class CompletionOverrideLookup : public swift::VisibleDeclConsumer {
     bool missingOverride = Reason == DeclVisibilityKind::MemberOfSuper &&
                            !isKeywordSpecified("override");
 
-    if (missingDeclIntroducer && missingAccess)
-      Builder.addAccessControlKeyword(
-          std::min(VD->getFormalAccess(), AccessibilityOfContext));
+    if (missingDeclIntroducer && missingAccess) {
+      auto Access = std::min(VD->getFormalAccess(), AccessibilityOfContext);
+      // Never emit 'private' for override or conformance decl.
+      if (Access == Accessibility::Private)
+        Access = Accessibility::FilePrivate;
+      Builder.addAccessControlKeyword(Access);
```

Comment:
@benlangmuir @jrose-apple 
Maybe, we might want to change this to:
```c++
if (Access >= Accessibility::Public)
  Builder.addAccessControlKeyword(Access);
```
Because
```
private C : P {
   fileprivate func foo() {}
}
```
is roughly equivalent to:
```
private C : P {
   func foo() {}
}
```
And the code looks simple.